src/components/ui/Badge/Badge.module.scss

```diff
@@ -0,0 +1,30 @@
+.badge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--color-orange);
```

Comment:
we should get `background-color` from `.primary` or `.secondary`, right? Maybe we could remove it from there, wdyt?

Reply:
Oh right, I completely overlooked that